charts/argo-cd/Chart.yaml

```
@@ -21,4 +21,4 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Fixed]: Correct ArgoCD applicationset controller port"
+    - "[Fixed]: Remove Argo CD notification default notifier"
```

```suggestion
    - "[Removed]: Remove Argo CD notification default notifier"
```

Let's use the correct keyword so everyone is informed that something is gonna removed